mantaray/received.py

```diff
@@ -228,6 +234,20 @@ def _handle_quit(server_view: views.ServerView, nick: str, args: list[str]) -> N
             view.userlist.remove_user(nick)
 
 
+def _handle_away(server_view: views.ServerView, nick: str, args: list[str]) -> None:
+    for view in _get_views_relevant_for_nick(server_view, nick):
+        if not args:
+            view.add_message(
+                "*", (nick, ["other-nick"]), (" is no longer away.", ["info"])
```

Comment:
I no longer think this is a good idea. It will result in a lot of spam in a big channel, and doesn't solve an actual problem in small channels. But we should update the userlist colors.

Also, I recently changed how `add_message()` should be called, so this needs some updating :)